src/context.cc

```diff
@@ -4487,7 +4489,8 @@ void Context::displayPomodoro() {
                                              "",  // project_guid
                                              "pomodoro-break",
                                              0,
-                                             0);  // tags
+                                             0,
+                                             true);  // tags
```

Comment:
The `//tags` comment is misleading. Please move it to the correct place or just remove it.